src/components/TestRail/TestRail.js

```diff
@@ -174,15 +182,52 @@ class TestRail {
      *
      * @param {string} runID
      * @param {Result[]} testResults
+     * @param includeCasesOnlyIfExistInRun
      * @returns {Promise<AxiosResponse<*>>}
      */
-    sendBatchResults(runID, testResults) {
+    async sendBatchResults(runID, testResults, includeCasesOnlyIfExistInRun) {
+        if (typeof includeCasesOnlyIfExistInRun !== 'boolean') {
+            includeCasesOnlyIfExistInRun = true; //avoid crashes due to nonexistent cases
```

Comment:
If you want to be backward compatible, this should be `false`
```suggestion
            includeCasesOnlyIfExistInRun = false; //avoid crashes due to nonexistent cases
```